src/pages/home/release-notes/index.md

```diff
@@ -7,6 +7,66 @@ Keywords:
 
 # Release notes
 
+## September 4, 2024
+
+### iOS Notification Content Extension 5.1.0-beta
```

Comment:
i guess this should say `AEPSwiftUI` framework - not Notification Content Extension

Reply:
Will make the changes on `main` after merging this in.